serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc

```diff
@@ -130,28 +131,20 @@ __  ____  __  _____   ___  __ ____  ______
 
 [NOTE]
 ====
-Note that the URI was translated to `http://greeting-quarkus-cli.serverless-workflow-greeting-quarkus.10.99.154.147.sslip.io` when the application started, and will be used at runtime any time it needs it.
+In the previous example, the URI is translated to `http://greeting-quarkus-cli.serverless-workflow-greeting-quarkus.10.99.154.147.sslip.io` when the application started. The translated URI is used at runtime, when needed.
 ====
-== Discovery Precedence
-
-Depending on the resource to be discovered, there are some paths to follow, please see the following image for more details:
-
-image::cloud/sw-discovery-flow.jpg[]
-
 
-== Startup time: discovery enabled x disabled
-
-As the Service Discovery scans the Quarkus Configurations during the startup, it can cause a small delay as shown below:
+The Kubernetes service discovery scans the Quarkus configuration during the startup, which can also cause small delay as follows:
 
 [tabs]
 ====
-Service Discovery Enabled::
+Kubernetes service discovery is enabled::
 +
 [source,shell]
 ----
 (main) sw-service-discovery 1.0.0-SNAPSHOT on JVM (powered by Quarkus 999-SNAPSHOT) started in 2.360s. Listening on: http://0.0.0.0:8080
 ----
-Service Discovery Disabled::
+Kubernetes service discovery is disabled::
```

Comment:
same

Reply:
I think the sentence looks better with is.